### PR TITLE
AV-171818 Fixing static route population

### PR DESCRIPTION
**Pull request by aaha97**

### CODE REVIEW

File: internal/lib/lib.go [hunk 1]
```diff
@@ -1959,19 +1959,52 @@ func IsChanClosed(ch <-chan struct{}) bool {
 	return false
 }
 
-func GetIPFromNode(node *v1.Node) (string, string) {
-	var nodeV4, nodeV6 string
```
Comment (akshayhavile) [inline internal/lib/lib.go:1963]: why did we remove this and return statement?

**Review by swathinsankaran — APPROVED** [review]
LGTM

File: internal/lib/lib.go (hunk 1 above)
Comment (aaha97) [reply internal/lib/lib.go]: it was just a choice to use named return values.
i have changed it back to improve readability.

**Comment by aaha97** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Comment by akshayhavile** [comment]
@aaha97: Once you complete testing, please merge it.

**Comment by swathinsankaran** [comment]
@aaha97, @akshayhavile Are we planning to merge this PR?

**Pull request closed without merging**